### ISSUE TITLE
Unable to copy/move to SD card

### ISSUE THREAD

**Issue by touzainanboku** [UnevenSoftware/LeafPic#246]
Currently you can't really move or copy pictures from normal folders to any other directory on the SD card that isn't part of the "local folders" (including hidden folders, even if they do have pictures in them), since you can't browse the SD card in the menu that pops up when you want to copy or move pictures.

**Comment by dnldsht** [comment]
duplicate of #201

**Comment by touzainanboku** [comment]
Not really, I'm not saying moving doesn't work, I'm saying you can't select folders on the SD card that currently don't have pictures in them, or are hidden folders. Different issues. 

Unless you meant that they're related and you want them to be a single issue. In which case you're right.
